AK/StdLibExtras.h

```diff
@@ -134,6 +134,18 @@ using RawPtr = typename Detail::_RawPtr<T>::Type;
 
 }
 
+// TODO: Make this work for more than 1 parameter.
+// TODO: Make this not a macro?
```

Comment:
Could try
```c++
template<typename Func, typename... Args>
static inline bool do_callback(Func&& func, Args&&... args) {
    if constexpr (IsVoid<decltype(func (args...))>) {
        func (forward<Args>(args)...);
        return false;
    } else {
        return func(forward<Args>(args)...) == IterationDecision::Break;
    }
}
```

and using it like `if (do_callback(func, args...)) return;`

(Note: on phone, expect typos and syntax errors :P)

Reply:
Thank you, I changed the implementation and documented the meaning of the return value.